app/javascript/packs/validators/validators_map.vue

```diff
@@ -0,0 +1,119 @@
+<template>
+  <div class="card map">
+    <div class="map-background">
+      <div class="map-points">
+        <!-- orientation points - TODO to remove after all map-related tasks are done -->
+        <!--
+        <div class="map-point map-point-sm" title="Map Center" style="left: 50%; bottom: 50%">X</div>
+        <div class="map-point map-point-md" title="Point 0,0"
+             :style="{ left: position_horizontal(0),
+                       bottom: position_vertical(-0) }">00</div>
+        <div class="map-point map-point-sm" title="Sydney"
+             :style="{ left: position_horizontal(147.1201174),
+                       bottom: position_vertical(-33.0996337) }">S</div>
+        <div class="map-point map-point-lg" title="Sth America End"
+             :style="{ left: position_horizontal(22.674129),
+                       bottom: position_vertical(-34.166060) }">SAF</div>
+        -->
+
+        <div v-for="data_center in data_centers"
+             :class="set_map_point_class(data_center.validators_count)"
+             :style="{ left: position_horizontal(data_center.longitude),
+                       bottom: position_vertical(data_center.latitude) }"
+             :title="data_center.data_center_key"
+             v-on:click="show_data_center_details(data_center)">
+          {{ data_center.validators_count }}
+        </div>
+      </div>
+    </div>
+
+    <div class="map-legend">
+      <div class="map-legend-col">
+        <small class="text-muted">Total in {{ network }}</small>
+        <div class="small fw-bold">
+          <strong class="text-success">{{ total_validators_count }}</strong> Validators
+        </div>
+        <div class="small">
+          <strong class="text-success">{{ total_nodes_count }}</strong> RPC Nodes
+        </div>
+      </div>
+      <div ref="dataCenterDetails" class="map-legend-col invisible">
```

Comment:
perhaps there could be something like "click on a node to see the details" when no node is selected